streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBSessionStore.java

```diff
@@ -41,27 +40,26 @@
                         final Serde<AGG> aggSerde) {
         super(bytesStore);
         this.keySerde = keySerde;
-        this.bytesStore = bytesStore;
         this.aggSerde = aggSerde;
     }
 
     @Override
     @SuppressWarnings("unchecked")
     public void init(final ProcessorContext context, final StateStore root) {
-        final String storeName = bytesStore.name();
+        final String storeName = name();
         topic = ProcessorStateManager.storeChangelogTopic(context.applicationId(), storeName);
 
         serdes = new StateSerdes<>(
             topic,
             keySerde == null ? (Serde<K>) context.keySerde() : keySerde,
             aggSerde == null ? (Serde<AGG>) context.valueSerde() : aggSerde);
 
-        bytesStore.init(context, root);
+        super.init(context, root);
     }
 
     @Override
     public KeyValueIterator<Windowed<K>, AGG> findSessions(final K key, final long earliestSessionEndTime, final long latestSessionStartTime) {
-        final KeyValueIterator<Bytes, byte[]> bytesIterator = bytesStore.fetch(
+        final KeyValueIterator<Bytes, byte[]> bytesIterator = wrappedStore().fetch(
```

Comment:
This is ugly -- we should have access to the wrapped store directly via inheritance. (also below)